src/main/java/core/basesyntax/strategy/BiFunctionSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.impl.AddBiFunction;
+import core.basesyntax.strategy.impl.SubtractBiFunction;
+import java.util.Map;
+import java.util.function.ToIntBiFunction;
+
+public class BiFunctionSupplier {
+    private Map<Operation, ToIntBiFunction<Integer, Integer>> operations;
+
+    public BiFunctionSupplier() {
+        this.operations = Map.of(Operation.BALANCE, new AddBiFunction(),
```

Comment:
balance operation should replace value, not be added to current

Reply:
not fixed